.github/workflows/build-wheels.yml

```
@@ -0,0 +1,131 @@
+name: Build OFRAK Wheels
+
+# Trigger on push to feature/windows_tests branch
+on:
+  push:
+    branches:
+       - master
+#      - feature/windows_tests
+  workflow_dispatch:
+
```

```suggestion

defaults:
  run:
    shell: bash -euxo pipefail {0}

```

This will ensure more verbose output, that it uses bash features instead of POSIX sh, and that it fails fast if any of the individual commands fail.